Airports/EGPC/Runway.txt

```diff
@@ -1 +1 @@
-13  31  129 309 N058.27.49.800 W003.06.18.000 N058.27.13.800 W003.04.49.800
\ No newline at end of file
+13  31  129 309 N058.27.44.520 W003.06.07.610 N058.27.19.630 W003.05.04.530 EGPC Wick
```

Comment:
you can remove the text from the end - the compiler adds this in for us

Reply:
Resolved in branch